README.md

```diff
@@ -87,9 +95,9 @@ There are three types of inference functions supported by Visual Blocks:
 
 1. **generic**: Generic inference functions accept input tensors and return
     output tensors.
-1. **text_to_text**: Text to text inference functions accept strings and return
+2. **text_to_text**: Text to text inference functions accept strings and return
```

Comment:
Actually using 1. 1. 1. .. is the recommended way. See https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#lists

The advantage is that you don't need to change these numbers when you delete or add an item in the list

Reply:
Fixed in the new commit.